cluster-service/deploy/openshift-templates/arohcp.devenv.md

```diff
@@ -1,184 +0,0 @@
-# How to run CS in DemoMode on AKS
```

Comment:
Is this obsolete?

Reply:
Nah, just deleted by mistake.  I added it back.